scene/animation/animation_player.cpp

```diff
@@ -977,6 +978,7 @@ void AnimationPlayer::remove_animation(const StringName &p_name) {
 
 	stop();
 	_unref_anim(animation_set[p_name].animation);
+	animation_set[p_name].animation->set_animation_name("");
```

Comment:
Does it make sense to reset the animation name before deleting it?

Reply:
The Animation resource doesn't necessarily gets deleted after this. That's why I thought it would be best to reset the name.

Reply:
Ok, sounds good :)